### PR TITLE
[Feature] Fully support DiffusionDet in projects

### PR DESCRIPTION
**Pull request by BIGWangYuDong**

### CODE REVIEW

File: projects/DiffusionDet/README.md [hunk 1]
```diff
@@ -60,14 +60,30 @@ We give a table to compare the inference results on `ResNet50-500-proposals` bet
...
+```
 
 ### Testing commands
 
 In MMDetection's root directory, run the following command to test the model:
 
 ```bash
-python tools/test.py projects/DiffusionDet/configs/${CONFIG_PATH} ${CHECKPOINT_PATH}
+# for 1 step inference
+export CONFIG_FILE=projects/DiffusionDet/configs/diffusiondet_r50_fpn_500-proposals_1-step_crop-ms-480-800-450k_coco.py
```
Comment (RangiLyu) [inline projects/DiffusionDet/README.md:81]: Can we use `--cfg-options model.bbox_head.sampling_timesteps=4` to control steps? It's much simpler.

File: projects/DiffusionDet/README.md [hunk 2]
```diff
@@ -60,14 +60,30 @@ We give a table to compare the inference results on `ResNet50-500-proposals` bet
 
 ### Training commands
 
-MMDetection currently does not fully support training DiffusionDet.
+In MMDetection's root directory, run the following command to train the model:
+
+```bash
+python tools/train.py projects/DiffusionDet/configs/diffusiondet_r50_fpn_500-proposals_1-step_crop-ms-480-800-450k_coco.py
```
Comment (RangiLyu) [inline projects/DiffusionDet/README.md:66]: Need to tell users that the 4-step config and 1-step config have no difference during training.

File: projects/DiffusionDet/README.md (hunk 1 above)
Comment (BIGWangYuDong) [reply projects/DiffusionDet/README.md:81]: So we can delete 4 step config file?

Comment (Li-Qingyun) [reply projects/DiffusionDet/README.md:81]: > So we can delete 4 step config file?

I agree with deleting 4 step config!

**Comment by BIGWangYuDong** [comment]
> AspectRatioBatchSampler

Hi @Li-Qingyun,
you can find a setting in _base_/datasets/cocodetection.py:
```
batch_sampler=dict(type='AspectRatioBatchSampler'),
```

**Comment by Li-Qingyun** [comment]
Okkkk, Thanks for your reply~  @BIGWangYuDong

**Review by RangiLyu — APPROVED** [review]

**Pull request merged**